src/components/modals/SaveAsSnippetModal.js

```
@@ -0,0 +1,162 @@
+        setContent('');
+      }
+      setKeyword('');
+      setSelectedFolderId('inbox'); // Default to inbox
+      setError('');
+
+      // Fetch snippet folders
+      window.electron.invoke('get-snippet-folders')
+        .then(folders => {
+          if (Array.isArray(folders)) {
+            setAvailableFolders(folders);
+          } else if (folders && folders.error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)